src/components/autosuggest/autosuggest.ui.js

```diff
@@ -497,13 +504,12 @@ export default class AutosuggestUI {
   }
 
   createWarningElement(content) {
-    const warningListElement = document.createElement('li');
+    const warningDivElement = document.createElement('div');
```

Comment:
I think this should be renamed because its not very descriptive maybe `warningContainer` or something